test/parallel/test-http-pipeline-flood.js

```diff
@@ -53,6 +53,15 @@ function parent() {
     const args = [__filename, 'child', this.address().port];
     const child = spawn(process.execPath, args, { stdio: 'inherit' });
     child.on('close', common.mustCall(function() {
+      if (common.isIBMi) {
+        // On IBM i calling server.close after the child process closes
+        // is not enough to end the server. For some reason the connection still
+        // lingers around with the server even though the process was closed.
+        // Calling closeAllConnections cleans up the lingering connection.
+        // Then calling server.close stops the server from listening
+        // for new connections and ends the server.
+        server.closeAllConnections();
```

Comment:
I think this invalidates the test. The child process gets killed after 200 ms and that should result in the TCP connections getting terminated. Does that happen on ibm i?

Reply:
The test expects the operating system to clean up the child process's TCP connections when it's terminated. That in turn should cause the TCP connections in the parent process to hang up with e.g. a ECONNRESET error.

If that doesn't happen, then the problem is probably somewhere lower in the stack.

Reply:
If you mean like a getsockopt() option: no.